portal/templates/flask_user/reset_password.html

```diff
@@ -6,10 +6,13 @@ <h1>{%trans%}Reset Password{%endtrans%}</h1>
 
 <form action="" method="POST" class="form" role="form">
     {{ form.hidden_tag() }}
-    {{ render_field(form.new_password, data_error="Oops, the password does not meet the minimum requirements.", pattern="^(?=.*[a-z])(?=.*[A-Z])(?=.*\d).{8,}$",
-    infoText="Password must have at least 8 characters with one lowercase letter, one uppercase letter and one number", tabindex=10) }}
+    {% set password_requirement_text = gettext('Password must have at least 8 characters with one lowercase letter, one uppercase letter and one number') %}
```

Comment:
Unless the language has already been approved, could you use `1` instead of `one` to be consistent with `8` (or switch `8` to `eight`)?